src/services/expenses.service.js

```diff
@@ -0,0 +1,100 @@
+const { getUserById } = require('./users.service');
+const expenses = [];
+
+function getAll() {
+  return expenses;
+}
+
+function getExpensesByUserId(userId) {
+  return expenses.filter((e) => e.userId === parseFloat(userId));
+}
+
+function getExpenseByQuery(query) {
```

Comment:
what happen if I have all 4 properties available? Is it a case that we should verify?

`to, from, userId, categories`

As an example: can I get the to and from expenses for a specific user?